cdk/lib/adx-data-plane-stack.ts

```diff
@@ -0,0 +1,146 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Completely just out of curiosity is there a benefit of having CDK code on `.ts` rather than on python? 
Is `.ts` the preferred cdk language?

Reply:
No this was just preference on my end. Was also thinking if we end up using CDK more, we could potentially create shared node modules for similar resource setups etc.